index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TEDxUniversity</title>
+    <!-- link to external resources -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <!-- link to other files -->
+    <link rel="stylesheet" href="css/style.css" />
+    <script src="js/main.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <div class="d-navbar flex g-padding">
+        <a href="#">
+          <img class="logo" src="images/tedx-logo.jpg" alt="TEDx's Logo" />
+        </a>
+        <nav class="d-nav">
+          <ul class="menu flex">
+            <li><a class="menu-item" href="/html/about.html">About</a></li>
```

Comment:
- [ ] Link to about page is also broken please be sure you fixed these links also.(Mobile and Desktop)

```suggestion
            <li><a class="menu-item" href="./html/about.html">About</a></li>
```